### PR TITLE
Use latest pull request info

### PR DESCRIPTION
**Pull request by mforutan**
The pull request info on the context isn't up to date. When a user updates the title and re-runs the workflow, it would be outdated. Therefore fetch the pull request via the REST API to ensure we use the current title.

### CODE REVIEW

**Comment by rdbartram** [comment]
looking forward to this PR getting in...please merge it up asap

File: index.js [hunk 1]
```diff
@@ -20,7 +20,22 @@ async function run() {
             return;
         }
 
-        const title = github.context.payload.pull_request.title;
+        const owner = github.context.payload.pull_request.base.user.login;
+        const repo = github.context.payload.pull_request.base.repo.name;
+
+        const client = new github.GitHub(process.env.GITHUB_TOKEN);
```
Comment (Kevmo92) [inline index.js:26]: I think you could use just `github.token` instead. Doing so should remove the need to set it in the env as well.
```suggestion
        const client = new github.GitHub(github.token);
```
https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

**Comment by mforutan** [comment]
@deepakputhraya, thank you for taking care of the change, can you please merge this now?

**Comment by mforutan** [comment]
@deepakputhraya, something is not right, I am getting `Bad credentials` with the latest code

**Comment by mforutan** [comment]
I think the correct way to do this to still use the token from env variables but give it a default like this one: https://github.com/actions/checkout/blob/28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b/action.yml#L12-L24

I don't believe that we can use the env variables from the GitHub context in the code like that

**Comment by deepakputhraya** [comment]
Updated the changes and merged them here - #10

**Pull request closed without merging**